_posts/2024-09-02-sr2025-registration-open.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Registration opens for Student Robotics 2025
+---
+
+{% include figure.html src="/images/content/blog/sr2024/sr2024-photo.jpg"
+           caption="All the teams from SR2024" %}
+
+We're excited to announce that registration for the 2025 season of Student
+Robotics is now open!
+
+Based in the UK, Student Robotics challenges teams of 16 to 19 year-olds to
+design, build and program fully autonomous robots to compete in our annual
+competition. Teams will have just six months to engineer their creations. As
+well as supplying teams with a kit, which they can use as a framework for their
+robot, we mentor the teams over this period. Thanks to the generosity of our
+sponsors, we provide all of this to our teams at no cost.
+
+The competition cycle will start with an in-person [Kickstart event][kickstart]
+hosted at the University of Southampton on October 12th. During the event the game and the
+structure of the competition will be announced.
+
+The competition year will culminate in an in-person competition over two days in
+around Easter 2025, which will see the robots compete through a league stage and
+a seeded knockout. As usual the prizes will recognise not only the teams which
+come top in the knockouts, but also those who excel in other ways.
```

Comment:
Slight wording tweaks?
```suggestion
The competition year will culminate with an in-person competition over two days 
around Easter 2025. This will see the robots compete through a league stage and
a seeded knockout. As usual the prizes will recognise not only the teams which
come top in the knockouts, but also those who excel in other ways.
```